### ISSUE TITLE
Wave beam dealing inconsistent damage depending on whether the beam comes from the top or the bottom

### ISSUE THREAD

**Issue by Miepee** [MetroidAdvRandomizerSystem/MARS-Fusion#48]
According to herc, when the beam leaves the cannon going down it freezes and when it leaves the cannon going up it doesnt, which is bad for logic, as it means that it can kill enemies instantly which were expected to get frozen.

**Comment by AntyMew** [comment]
Yeah, it goes without saying this is a bug. I'm not sure why the behavior varies depending on the direction of the beam, but regardless I'm pretty sure it has something to do with multiple projectiles hitting an enemy at the same time

**Comment by Miepee** [comment]
This issue is actually related to wave beam, not ice beam!
If you try to kill kihunters with wave only, depending on how you shoot the kihunters, you will need anywhere between 5 and 11 shots.

**Comment by zarakava** [comment]
In beam-stacking.s, there is a comment that calls out to fire two projectiles with Wave Beam (uncharged, W/O Wide Beam)

![Image](https://github.com/user-attachments/assets/da507be3-8d80-47e2-b06d-d00e844608a5)

On review of the code and logic, the current implementation is at odds with the comment, and attempts to fire one projectile, but also unintentionally fires a second projectile on the "High Shot" of the Wave Sine.

To fix this issue, based on discussion in the Randovania Discord, the code will correctly fire 2 projectiles, one High and one Low
